exe/traker

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require File.expand_path(File.join(Dir.pwd, 'config/environment'))
+require 'optparse'
+require 'traker'
+
+# options = {}
+
+subtext = <<HELP
+  Possible commands are:
+     list:  lists rake tasks known to Traker (shows list of pending tasks by default)
+  See 'traker COMMAND --help' for more information on a specific command.
+HELP
+
+global = OptionParser.new do |opts|
+  opts.banner = 'Usage: traker [options] [subcommand] [options]'
+  # opts.on('-v', '--[no-]verbose', 'Run verbosely') do |v|
```

Comment:
`-v` is often associated with `--version`. I would suggest adding `-d/--debug` for this purpose.

Reply:
sure, I just left this as an example how we can add flags